mkdocs.yml

```diff
@@ -96,17 +96,17 @@ nav:
           - Parallel Processing Python: _user_guide/best_practices_parallel_processing_python.md
       - R:
           - Best Practices in R: _user_guide/best_practices_r.md
-      - Julia:
-          - Running Julia Interactively: _user_guide/julia.md
-      - Optimization Software:
-          - Optimization: _user_guide/best_practices_optimization.md
+      - Julia: _user_guide/julia.md
+      - SAS: _user_guide/sas.md          
+      - Optimization Software: _user_guide/best_practices_optimization.md
+      - Data:
+          - Storage: _user_guide/storage.md
+          - Data Transfer: _user_guide/data_transfer.md      
       - Troubleshooting:
           - Why isn't JupyterHub Working?: _user_guide/best_practices_jupyter_does_not_work.md
           - Man-in-the-Middle Warning: _user_guide/man_in_middle.md
-          - Julia on Jupyter: _user_guide/best_practices_julia_jupyter.md
```

Comment:
You want to nuke this page from User Guide? Or move it somewhere?

Reply:
Yeah, it's not 'troubleshooting' the way the other ones are.  It should probably just go on a general jupyter or a general julia page?

Reply:
yeah, maybe we can have two julia pages, general julia and this page nested under julia